examples/keyed_column/src/main.rs

```diff
@@ -0,0 +1,60 @@
+use iced::widget::{Column, row, button, keyed_column, text, column, container};
+
+
+pub fn main() -> iced::Result {
+    iced::run("A keyed_column example", Example::update, Example::view)
+}
+
+#[derive(Default)]
+struct Example;
+
+#[derive(Debug, Clone, Copy)]
+enum Message {
+    Nothing,
+}
+
+impl Example {
+    fn update(&mut self, message: Message) {
+        //
+    }
+
+    fn view(&self) -> Column<Message> {
+        let items = vec![
+            (
+                1, // key
+                container( // widget
+                    row![
+                        text("Item 1"),
+                        text("Description of item 1"),
+                        button("My button").on_press(Message::Nothing)
+                    ].spacing(30).padding([10, 0, 0, 0])
+                ).into()
+            ),
+            (
+                2, // key
+                container( // widget
+                    row![
+                        text("Item 2"),
+                        text("Description of item 2"),
+                        button("My button").on_press(Message::Nothing)
+                    ].spacing(30).padding([10, 0, 0, 0])
+                ).into()
+            ),
+            (
+                3, // key
+                container( // widget
+                    row![
+                        text("Item 3"),
+                        text("Description of item 3"),
+                        button("My button").on_press(Message::Nothing)
+                    ].spacing(30).padding([10, 0, 0, 0])
+                ).into()
+            )
+        ];
+
+        column![
+            text("My itens with keys").size(30),
```

Comment:
Typo

Reply:
Do you mean it has spelling errors? I can't put my finger on it, English isn't my native language, sorry.

Reply:
Yes. Shouldn't it be "My items with keys"?